src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -4433,6 +4439,12 @@ To prevent NuGet from downloading packages during build, open the Visual Studio
   <data name="FileConflictChoiceText_cht" xml:space="preserve">
     <value>[Y] 是  [A] 全部皆是  [N] 否  [L] 全部皆否 ?</value>
   </data>
+  <data name="Error_InvalidSolutionDirectory_deu" xml:space="preserve">
```

Comment:
I think it's preferable if you don't add the localization yourself. 

Maybe @dominoFire can help get it reviewed by localization folks, but in general it's something something that may slow down reviews.

Reply:
@papeh Can you please revert the localization versions of the strings? That way we can move forward toward merging this.